task/task.go

```diff
@@ -0,0 +1,42 @@
+package task
+
+import (
+	"errors"
+	"log"
+
+	"github.com/symflower/eval-dev-quality/language"
+)
+
+var (
+	// ErrTaskUnsupported indicates that a task is unsupported.
+	ErrTaskUnsupported = errors.New("task unsupported")
+)
+
+// Identifier holds the identifier of a task.
+type Identifier string
+
+// AllIdentifiers holds all available task identifiers.
+var AllIdentifiers []Identifier
+
+// registerIdentifier registers the given identifier and makes it available.
+func registerIdentifier(name string) (identifier Identifier) {
+	identifier = Identifier(name)
+	AllIdentifiers = append(AllIdentifiers, identifier)
+
+	return identifier
+}
+
+var (
+	// IdentifierWriteTests holds the identifier for the "write test" task.
+	IdentifierWriteTests = registerIdentifier("write-tests")
```

Comment:
Not sure about the naming. I think before it was called "generate tests", I just noticed that in the end.

Reply:
write tests is fine with me. Since the POV is that we are benchmarking a developer.

Just to note this: in the LLM world we usually talk about CodeGen